src/typescript/08-classes/classes.md

```diff
@@ -252,91 +183,106 @@ npm run 4a-classes
 <details>
 <summary>Solution</summary>
 
-✏️ Update `4a-classes-hello-dino.ts` to the following:
+✏️ Update  **hello-dino.ts** to look like:
 
-```typescript
-class DinoKeeper {
-  name: string;
+@diff ../../../exercises/typescript/08-classes/01-problem/src/classes/hello-dino.ts ../../../exercises/typescript/08-classes/01-solution/src/classes/hello-dino.ts
+@highlight 1-2, 4, 8-9,11, 16
 
-  constructor(name: string) {
-    this.name = name;
-  }
+</details>
 
-  sayHi(): string {
-    return `${this.name} says "hi"`;
-  }
-}
-let employee1 = new DinoKeeper('Joe');
-employee1.sayHi();
-//Logs "Joe says "hi""
+## Objective 2: Extend a class
 
-export default DinoKeeper;
-```
+### Inheritance
 
-</details>
-@highlight 1-2, 4, 8-9,11, 16
+Inheritance is a way to extend functionality of existing classes. If the derived class contains its own constructor function, it MUST call a super method with params matching that of its parent class. 
 
-## Exercise: Extend a Class
+Super is a call to the parent constructor method to ensure the properties are set for the parent. The following shows accessing the move method from the parent class and adding run and talk methods to the child class.
 
-### The Problem
+@sourceref ./6-5-inheritance.ts
+@codepen
+@highlight 9,11
 
-Edit `4b-specialist.ts` to write a new `Specialist` class:
+### Public modifier
 
-```ts
-import DinoKeeper from "./4a-classes-hello-dino";
+In TypeScript all members are public by default, meaning they are publicly accessible.
```

Comment:
`TypeScript all` -> `TypeScript, all`

Also, “are publicly accessible” is kinda vague here—I would say something like “can be accessed from outside the class.”